.github/workflows/performance.yml

```diff
@@ -69,13 +69,13 @@ jobs:
             - name: Compare performance with base branch
               if: github.event_name == 'push'
               # The base hash used here need to be a commit that is compatible with the current WP version
-              # The current one is 9725060a5b18904c6cc5fdbe4b06fbde7419e02c and it needs to be updated every WP major release.
+              # The current one is 5f4c9c853b15092ed885d5280edefb973c37d9e9 and it needs to be updated every WP major release.
```

Comment:
Just checking, what does "compatible with the current WP version" mean here?

Shipped with that version, or just works with it?

Is https://github.com/WordPress/gutenberg/commit/5f4c9c853b15092ed885d5280edefb973c37d9e9 was committed yesterday.

Reply:
> A simple way to choose commit is to pick a very recent commit on trunk with a passing performance job.

https://developer.wordpress.org/block-editor/explanations/architecture/performance/#update-the-reference-commit

Reply:
Yes, this one works, it has values for all the metrics.